api/routes_lens.go

```diff
@@ -68,5 +68,9 @@ func (api *API) submitSearchRequest(c *gin.Context) {
 		api.LogError(err, eh.FailedToSearchError)(c, http.StatusBadRequest)
 		return
 	}
+	if len(resp.Names) == 0 {
+		api.LogError(err, eh.NoSearchResultsError)(c, http.StatusBadRequest)
```

Comment:
I don't think this qualifies as an error in terms of logging - at least how I see it is when we log an error, its something kinda to very critical, that needs attention, whereas someone searching "asdfjalsef" and turning up no results isn't something we need to be immediately aware of

Reply:
we might want to do an info-level log maybe, since it might be useful for watching search trends

Reply:
Good call